### ISSUE TITLE
nr.gz database error message during zcat step in download_DB.sh

### ISSUE THREAD

**Issue by mhyleung** [functional-dark-side/agnostos-wf#14]
Dear all

We have been encountering issues processing the nr database with the following outputs:

```
aria2c --file-allocation=none -c -x 10 -s 10 ftp://ftp.ncbi.nih.gov/blast/db/FASTA/nr.gz

${MMSEQS}" createdb nr.gz nr.db --write-lookup 0 -v 0
nr.db exists and will be overwritten.
createdb nr.gz nr.db --write-lookup 0 -v 0

MMseqs Version:         2f1db01c5109b07db23dc06df9d232e82b1b4b99-MPI
Database type           0
Shuffle input database  true
Createdb mode           0
Write lookup file       0
Offset of numeric ids   0
Compressed              0
Verbosity               0

Converting sequences
[158056214] 16m 8s 55msss
Time for merging to nr.db_h: 0h 6m 32s 806ms
Time for merging to nr.db: 0h 15m 7s 228ms
Database type: Aminoacid
Time for processing: 0h 41m 1s 161ms_

zcat nr.gz | grep '^>' | sed 's/^>//' | sed 's/ /\t/' | sed 's/ /_/g' | gzip > nr.proteins.tsv.gz
gzip: nr.gz: invalid compressed data--format violated

```

Seems like somehow the nr.gz file is not compressed properly or something? However if this was the case, does that mean that the mmseq step did not run properly? We did manage to complete the mmseq step without any error messages. Thanks

Marcus

**Comment by genomewalker** [comment]
Hi @mhyleung 

Most probably the file is corrupted and not properly extracted.

@ChiaraVanni can you add a validation that both checksums are the same?

The md5 is here ftp://ftp.ncbi.nih.gov/blast/db/FASTA/nr.gz.md5

This should be for all files we download if available

**Comment by mhyleung** [comment]
Thanks for the quick response.

So I simply ran `wget ftp://ftp.ncbi.nih.gov/blast/db/FASTA/nr.gz` and the md5 of the resulting `nr.gz` matches that of `ftp://ftp.ncbi.nih.gov/blast/db/FASTA/nr.gz.md5`

Whereas the nr.gz file I got from aria2c was not able to give me an md5, so most likely something was wrong with the previous file.

Is there a specific reason why aria2c was used specifically for this task? Thanks

Marcus
